File: Back To Sender Logistics/BackToSender.java
import java.util.Scanner;
import java.util.InputMismatchException;

public class BackToSender{

   static Scanner scanner = new Scanner(System.in);

   public static void main (String []args){

	int ridersWage = deliveryInput();
	System.out.print("The wage of the rider is: " + ridersWage);
   }

	private static int validateInput(){
	   int deliveryNumber = 0;
	   boolean isValid = false;
		while(!isValid){
			System.out.println("Enter colletion rate of successful delivery: ");
			
			try{
			   int number = scanner.nextInt();
				if(number < 0){
			   		System.out.print("Enter a valid input!!!");
				} else {
					deliveryNumber =  number;
				   isValid = true;
				}
			} catch (InputMismatchException e){
				System.out.print("Invalid input!!!. Enter a valid input!!!");
				scanner.next();
			}
		} 

		return deliveryNumber;
		
	}


	public static int deliveryInput(){

		int deliveryNumber = validateInput();
		int result;
				if (deliveryNumber < 50)
					result = deliveryNumber * 160 + 5000;
				else if (deliveryNumber <= 59 && deliveryNumber >= 50)
					result = deliveryNumber * 200 + 5000;
				else if (deliveryNumber <= 69 && deliveryNumber >= 60)
					result = deliveryNumber * 250 + 5000;
				else if (deliveryNumber >= 70)
					result = deliveryNumber * 500 + 5000;
				else
					result = 0;

		return result;

	}
}

File: Back To Sender Logistics/backToSender.py
def validate_input():
	while True:
		try:
			delivery_number = int(input("Enter number of successful delivery: "))
			if(delivery_number < 0):
				print("Enter a valid input!!!")
			else:
				return delivery_number
		except ValueError:
			print("Invalid input, enter a valid number!!!")

def delivery_input():
	delivery_number = validate_input()
	result = 0
	if delivery_number < 50:
		result = delivery_number * 160 + 5000
	elif delivery_number <= 59 and delivery_number >= 50:
		result = delivery_number * 200 + 5000
	elif delivery_number <= 69 and delivery_number >= 60:
		result = delivery_number * 250 + 5000
	elif delivery_number >= 70:
		result = delivery_number * 500 + 5000
	return result


result = delivery_input()
print("The wage of the rider is", result)

